src/maheshKJun20/NumbersExample/CheckArmStrongNumber.java

```diff
@@ -0,0 +1,25 @@
+package maheshKJun20.NumbersExample;
+
+public class CheckArmStrongNumber {
+	
+	void checkArmsStrongNum(int num) {
+		//Method to check and Display a number is Armstrong Number or not
+		int checkArmStrong = 0, temp = num;
+		while (temp>0) {
+			checkArmStrong += ((temp%10) * (temp%10) * (temp%10)) ;
```

Comment:
Suggestion:
You can take a look at power function - Math.pow(x,y)
which will help you achieve same result for multiplication.
i.e. Math.pow((number % 10), 3))